cachi2/core/errors.py

```diff
@@ -60,10 +60,10 @@ class InvalidInput(UsageError):
 
 
 class PackageRejected(UsageError):
-    """Cachi2 refused to process the package the user requested.
+    """Application refused to process the package the user requested.
```

Comment:
s/Application/The application/

(there are more instances in this same file)

Reply:
applied